src/compas_viewer/config.py

```diff
@@ -235,8 +235,18 @@ class StatusbarConfig(ConfigBase):
 @dataclass
 class SidebarConfig(ConfigBase):
     show: bool = True
-    sceneform: bool = True
-    items: list[dict[str, str]] = None
+    items: list[dict[str, str]] = field(
+        default_factory=lambda: [
+            {
+                "type": "Sceneform",
+                "action": {
+                    "Name": (lambda o: o.name),
+                    "Visible": (lambda o: o.show),
+                    "Locked": (lambda o: o.is_locked),
```

Comment:
Won't this conflict to #169 ? Here it is defining to simply show the value, but in other PR the behavior is binding them to toggles